generators/app/index.js

```diff
@@ -581,6 +581,13 @@ module.exports = yeoman.Base.extend({
         }
       }
     },
+    setLog4jdevProperties: function () {
+      if (this.bail) return;
+
+      // set the dev-log4j.properties file for the project package
+      var devLog4jPropertiesPath = 'repo/src/main/resources/alfresco/extension/dev-log4j.properties';
+      this.fs.write(devLog4jPropertiesPath, 'log4j.logger.' + this.projectPackage + '=${app.log.root.level}\n');
```

Comment:
nit: consider constructing the content in a variable prior to writing

The default file looks like this:
```
# You can add here custom log4j classes. This will be merged with the main log4j.properties
# See http://wiki.alfresco.com/wiki/Developing_an_Alfresco_Module#Best_Log4j_Configuration_Practices
log4j.logger.org.alfresco.demoamp.DemoComponent=${app.log.root.level}
```

This implementation seems to lose the comments, these comments are very important.